### DISCUSSION TITLE
how to troubleshoot ERR_TOO_MANY_REDIRECTS

### DISCUSSION THREAD

**Discussion by genfoch01** [NginxProxyManager/nginx-proxy-manager#3210] [General]
I deployed nginx proxy manager via docker-compose 

I set up my first host (bookstack ) like so : 
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/144620889/dc1ffb4d-8bb9-4c57-af61-52db3426ad99)

with an ssl cert (force ssh  and http/2 support ) 

works perfectly. 

so I have an old nas  (it does not support adding a cert: its old )  so I thought i'd use nginx in front of it to get rid of the annoying ssh warnings. 

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/144620889/46a63576-a70c-40f8-8e8a-1092b04a0b19)

I don't have ssl set up because I cant get the proxy to work. 

when I try to go though the proxy server I get 

 
![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/144620889/79cff609-f9f6-4687-959f-8c616107be68)

the access log shows : 
[22/Sep/2023:15:41:58 +0000] - 302 302 - GET https ilios.citystates.net "/admin" [Client 192.168.10.105] [Length 218] [Gzip -] [Sent-to 192.168.10.155] "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36" "-"

the fallback_error.log shows this;

2023/09/22 15:04:50 [error] 3577#3577: *22414 open() "/var/www/html/login" failed (2: No such file or directory), client: 192.168.10.105, server: localhost-nginx-proxy-manager, request: "GET /login HTTP/1.1", host: "dev2.citystates.net"

I am not sure how to attack this problem. It seems like it should be simple, i'm just not sure where to start. 

thanks for your time, 
GF

**Comment by sugarfunk** [comment]
Is cloudflare your name server? Make sure your ssl in CF is full or strict.

**Reply by genfoch01** [reply]
thanks for the response, sorry for the late reply.  I finally found the issue. I needed to add the APP_URL directive to the bookstack composer then everything started to work.
